src/components/QExpansionPanel/QExpansionPanel.vue

```diff
@@ -0,0 +1,95 @@
+<template>
+    <div class="rounded-lg q-panel border bg-white border-gray-200 shadow-lg shadow-black-500/50" :class="{ 'mb-2': state.panelIsOpen }" :data-panel-index="panelIndex">
+      <h2 class="accordion-header mb-0">
+        <button class="relative flex items-center w-full py-4 px-5 text-base text-gray-800 text-left border-0" type="button" @click="togglePanel">
+          <slot name="header"/>
+          <q-chevron :accent-color="props.accentColor" :state="state.panelIsOpen" class="ml-auto"/>
+        </button>
+      </h2>
+      <transition
+          :class="panelClasses"
+          @enter="enter"
+          @after-enter="afterEnter"
+          @leave="leave"
+      >
+        <div v-show="state.panelIsOpen">
+          <div class="slide__slot-container transition py-4 px-5">
+            <slot name="content"/>
+          </div>
+        </div>
+      </transition>
+    </div>
+</template>
+
+<script setup>
+import { reactive, computed } from 'vue';
+import QChevron from '../QChevron/QChevron.vue';
+
+const props = defineProps({
+  animated: {
+    type: Boolean,
+    default: false
+  },
+  panelIndex: {
```

Comment:
I think this props is useless.

Reply:
Same as <q-expansion-panels/>, it was for future improvements. Removing.